charts/argo-events/templates/eventbus-controller/deployment.yaml

```diff
@@ -45,9 +45,9 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: NATS_STREAMING_IMAGE
-              value: {{ .Values.eventbusController.natsStreamingImage }}
+              value: "{{ .Values.registry }}/{{ .Values.eventbusController.natsStreamingImage }}"
             - name: NATS_METRICS_EXPORTER_IMAGE
-              value: {{ .Values.eventbusController.natsMetricsExporterImage }}
+              value: "{{ .Values.registry }}/{{ .Values.eventbusController.natsMetricsExporterImage }}"
```

Comment:
Looks like these images are not on quay.io yet. Should I create a separate parameter, e.g. `natsRegistry` for that? The motivation for this change is that we have users hitting the DockerHub rate limit when pulling these images.

Reply:
Hmmm. I think we need at least a dedicated key to for the registry of the NATS images as these are external dependencies hosted on a different registry by default.

Maybe we could also leave it as-is and place a reminder into the README as overriding is already possible without this change.

Reply:
> overriding is already possible without this change

Could you elaborate?